pkg/controller/stacks/install/stackinstall.go

```diff
@@ -231,7 +232,8 @@ func (h *stackInstallHandler) update(ctx context.Context) (reconcile.Result, err
 	// TODO: should updates of the StackInstall be supported? what would that even mean, they
 	// changed the package they wanted installed? Shouldn't they delete the StackInstall and
 	// create a new one?
-	log.V(logging.Debug).Info("updating not supported yet", h.ext.GroupVersionKind(), fmt.Sprintf("%s/%s", h.ext.GetNamespace(), h.ext.GetName()))
+	groupversion, kind := h.ext.GroupVersionKind().ToAPIVersionAndKind()
```

Comment:
any reason not to just call `String()` on `h.ext.GroupVersionKind()`?
https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L161

Reply:
not a huge deal, just wondering what the reason to do more logic was